chsdi/tests/integration/__init__.py

```diff
@@ -32,6 +35,19 @@ def tearDown(self):
         del self.testapp
         del self.grids
 
+    @contextmanager
+    def getSession(self):
+        session = scoped_session(sessionmaker())
+        yield session
+        session.close()
+
+    def getRandomFeatureId(self, bodId):
+        models = models_from_bodid(bodId)
+        with self.getSession() as session:
+            query = session.query(models[0]).limit(1)
+            reslt = query.one()
```

Comment:
Yes, I had that once. Works fine with small datasets, explodes with big datasets :=), that's why I asked.

Truly random would be better, but can live with this. Maybe there's a way to randomize the first 1000 only?